src/coordination/include/nuraft/coordinator_cluster_state.hpp

```diff
@@ -61,8 +61,8 @@ struct CoordinatorInstanceState {
 void to_json(nlohmann::json &j, ReplicationInstanceState const &instance_state);
 void from_json(nlohmann::json const &j, ReplicationInstanceState &instance_state);
 
-using TRaftLog = std::variant<CoordinatorToReplicaConfig, std::string, utils::UUID, CoordinatorToCoordinatorConfig,
-                              InstanceUUIDUpdate>;
+using TRaftLog = std::variant<std::monostate, CoordinatorToReplicaConfig, std::string, utils::UUID,
```

Comment:
I don't think monostate is good here :joy: try to refactor this

Reply:
I am not sure what is wrong with monostate, we can discuss it on Monday but seems to me from cpp ref it is used that way as an empty tag?